source/clear-linux/get-started/bare-metal-install/bare-metal-install.rst

```diff
@@ -3,108 +3,32 @@
 Install Clear Linux host OS on bare metal
 #########################################
 
-These instructions will guide you through the installation of |CL| on bare
-metal using a :ref:`bootable USB drive<create-usb>`.
+This instruction set will guide you through the automatic installation of |CL|
+on bare metal using a bootable USB drive.
 
-Verify your system hardware is supported by |CLOSIA|. Review the
-:ref:`list of supported processors <supported-hardware>` and
-:ref:`instructions to run a compatibility check <compatibility-check>`.
+.. include:: ../compatibility-check.rst
+   :Start-after: compatibility-check:
 
-.. note::
-
-   The steps of this installation guide were tested using a NUC6i5SYH
-   Intel® NUC. Visit the `NUC6i5SYH product page`_ for detailed information.
-
-.. _create-usb:
-
-Create a Clear Linux bootable USB drive
-=======================================
-
-This procedure was created on an Ubuntu 16.04-based system where the USB
-drive is identified as :file:`/dev/sdb`. Make sure to map your correct USB
-device for this process. We recommend you use an 8GB USB drive or larger.
-Copying the |CL| image onto the USB drive formats the drive as a UEFI boot
-device. Therefore, the contents of the USB drive will be destroyed during the
-creation of the bootable USB drive. Make sure to save anything stored in the
-drive before proceeding.
-
-.. _download-clear-linux-image:
-
-Download the Latest Clear Linux Image
--------------------------------------
-
-Get the latest available |CL| installer image that you want to install
-to your system by using your web browser and downloading the latest
-:file:`clear-[release]-installer.img.xz` file from
-https://download.clearlinux.org/image/ where `[release]` is the release
-number of the current image that is available in this directory listing.
-
-This example uses release 10980 so we will download the
-:file:`clear-10980-installer.img.xz` image file and, optionally, the
-:file:`clear-10980-installer.img.xz-SHA512SUMS` file needed to verify the
-download.
-
-To verify the download, follow these steps:
-
-1. Go to the directory with the downloaded files.
-2. To verify the integrity of the file, enter the following commands:
-
-   .. code-block:: console
-
-      sha512sum ./clear-10980-installer.img.xz | diff ./clear-10980-installer.img.xz-SHA512SUMS -
+.. include:: ../bootable-usb/bootable-usb-linux.rst
```

Comment:
Why the multiple includes? Please try to keep complex markup as simple as possible. As it stands, you are adding all the content from bootable-usb-linux up to usb-next. This can be accomplished with a single include.

Reply:
We are using multiple includes to avoid duplicate labels. Our options were to remove the labels and duplicate content in other files that needed the "download-cl-image" label, or have multiple includes in this file. As per our conversation a few weeks ago, multiple includes were the best option. We have limited the labels as much as possible.

Reply:
Understood, thank you for the clarification.